unicodetools/src/main/java/org/unicode/props/UcdPropertyValues.java

```diff
@@ -62,8 +62,7 @@ public enum Age_Values implements Named {
         V12_0("12.0"),
         V12_1("12.1"),
         V13_0("13.0"),
-        V13_1("13.1"), // TODO: there is no Unicode 13.1, see
-        // https://github.com/unicode-org/unicodetools/issues/100
+        V13_1("13.1"),
```

Comment:
It looks like GenerateEnums + spotless removes this comment.

Reply:
Yes. Please restore it. Known issue.